packages/dialtone-vue2/.storybook/preview.jsx

```diff
@@ -1,5 +1,6 @@
 import '../css/dialtone-globals.less';
-import '@dialpad/dialtone-css/lib/dist/dialtone.css';
+import '@dialpad/dialtone-css';
+import 'overlayscrollbars/overlayscrollbars.css';
```

Comment:
Should we just include this in @dialpad/dialtone-css?

Reply:
It's possible yeah, but maybe it's better to include it in dialtone-vue css file as it is more related to a vue directive.

Reply:
ah yeah I agree with that

Reply:
Investigated a bit and in order to make the package as light as possible, it's better to keep it external and let the user include it only when they actually need it.

![Screenshot 2024-09-11 at 1 51 27 p m](https://github.com/user-attachments/assets/1b6e85b6-0752-4464-8c3a-7ea8e87f8993)

overlayscrollbars is 17kb and dialtone-vue styles are 54kb, embedding overlayscrollbars makes a 32% increase in size.

I know it doesn't sound too much as we're talking of Kb. But we're trying to make dialtone-vue as lightweight as possible, and maybe that external dependency makes it easier for the dialkits developers to use it in the shadow-root.